microprofile/fault-tolerance/src/main/java/io/helidon/microprofile/faulttolerance/FaultToleranceExtension.java

```diff
@@ -260,6 +264,19 @@ void registerFaultToleranceMetrics(@Observes AfterDeploymentValidation validatio
                 }
             });
         }
+
+        // Initialize executors for MP FT - default size of 16
+        io.helidon.config.Config config = io.helidon.config.Config.create();
+        FaultTolerance.scheduledExecutor(ScheduledThreadPoolSupplier.builder()
+                .threadNamePrefix("ft-mp-schedule-")
+                .corePoolSize(16)
```

Comment:
`16 is the default right?

Reply:
I think so. But defaults can change and a smaller number will likely result in FT TCK failures.